src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -297,6 +297,32 @@ const findPossibleSets = listOfExistingEntities => {
   }, [], listOfExistingEntities)
 }
 
+const getWorkflowInputSuggestionsForAttributesOfSetMembers = (selectedEntities, entityMetadata) => {
```

Comment:
Added some tests in [49e7346](https://github.com/DataBiosphere/terra-ui/pull/2832/commits/49e734642bf6d6b8373d2bffea7fcdfdf88629f3). I had to move the function to another file to get Jest to run (I think it couldn't handle the circular import libs/auth => components/common => components/icons => libs/auth).

Reply:
The tests look great ❤️ 

How would you feel about moving `getWorkflowInputSuggestionsForAttributesOfSetMembers` inside `src/libs/workflow-utils.js` instead of creating a dedicated file for it? Accordingly, the new test file could be moved to `src/libs/_tests/workflow-utils.test.js`.

Reply:
Moved in [47eee8c](https://github.com/DataBiosphere/terra-ui/pull/2832/commits/47eee8ce209d0cdf2739726e8961cbb0231440e9).